.buildkite/pipelines/on_merge.yml

```diff
@@ -7,13 +7,19 @@ steps:
     timeout_in_minutes: 10
     agents:
       queue: kibana-default
+    retry:
+      automatic:
+        - exit_status: '*'
+          limit: 1
 
   - wait
 
   - command: .buildkite/scripts/steps/on_merge_build_and_metrics.sh
-    label: Default Build and Metrics
+    label: Build Kibana Distribution and Plugins
     agents:
-      queue: c2-8
+      queue: c2-16
+    key: build
+    if: "build.env('KIBANA_BUILD_ID') == null || build.env('KIBANA_BUILD_ID') == ''"
```

Comment:
For the sake of metric baselines I wonder if we should have this option for this build...